### ISSUE TITLE
Digits are interpreted as integers in fixes or octals

### ISSUE THREAD

**Issue by phochste** [LibreCat/Catmandu#93]
E.g.

```
 set_fied(isbn,'0596002815');
```

will set isbn to '596002815'

And 

```
set_field(isbn,0596002815)
```

will throw an error:

```
Illegal octal digit '9' at (eval 633) line 1, at end of line
Illegal octal digit '8' at (eval 633) line 1, at end of line
```

**Comment by nics** [comment]
fixed in 1.2009

**Comment by vpeil** [comment]
thanks @nics
